QUEST_LV_0100.tsv

```diff
@@ -1509,60 +1509,60 @@ QUEST_LV_0100_20150317_001508	Actually I also Denoptic components, please look f
+QUEST_LV_0100_20150317_001545	$I see you came back safely. {nl} Well then, I must also prepare to leave.
+QUEST_LV_0100_20150317_001546	$What do we do? I think there are even more monsters now than when we first came. {nl} And the patients have not recovered yet. {nl}
+QUEST_LV_0100_20150317_001547	$Somehow I'll treat this man. Can you defeat the monsters around?
+QUEST_LV_0100_20150317_001548	$In order for these people to safely return to the village,{nl}we have to defeat these monsters.
+QUEST_LV_0100_20150317_001549	$Thank you. The patient also regained his consciousness. {nl} Though.. I think there is a problem.
+QUEST_LV_0100_20150317_001550	$This has to be because there is the evidence of salvation or something. {nl}I'm sure that's where the villagers were taken away to.
+QUEST_LV_0100_20150317_001551	$The Goddess must sent help. {nl}Follow the way on the right to find the Vubbe Outpost.
+QUEST_LV_0100_20150317_001552	$Thank you for saving me. {nl} Seems like you're the Revelator who's looking for evidence of salvation. {nl}
+QUEST_LV_0100_20150317_001553	$I was taken in late so I was left here, the others were all brought to the mine. {nl} I'll tell you the rest of the story in the Crystal Mine.
+QUEST_LV_0100_20150317_001554	$You came. {nl} First, let's clear the wagons blocking the way into the mines. {nl}
+QUEST_LV_0100_20150317_001555	$The wagons can be demolished with explosives that I have prepared. {nl}I'm concerned about the Vubbes that will come after hearing the explosion {nl}but with your skills, I think we'll be fine.
+QUEST_LV_0100_20150317_001556	$I'll let you take care of the Vubbes. {nl} I have other businesses to attend to inside. {nl}
+QUEST_LV_0100_20150317_001557	$Thank you. Thank you very much. {nl} You're a savior of our town. {nl}
+QUEST_LV_0100_20150317_001558	$Oh, have you seen a girl about 5 years of age? {nl} I heard she was together with them when the villagers were kidnapped.
+QUEST_LV_0100_20150317_001559	$Don't just stand there. {nl} Move if you want to help or go away.
+QUEST_LV_0100_20150317_001560	$Mines Manager
+QUEST_LV_0100_20150317_001561	$I'm hanging on somehow but I'm losing strength in my arms.
 QUEST_LV_0100_20150317_001562	$I should have been more careful. {nl} It's all my fault.
-QUEST_LV_0100_20150317_001563	I have my roots deep in the hunting area. {nl} Do not worry about me and better look after the others.
-QUEST_LV_0100_20150317_001564	$As the Mayor, I can't just leave the village like this. {nl} How could the Goddesses be so indifferent??
-QUEST_LV_0100_20150317_001565	$Aren't you a Revelator? The Goddess must've sent help. {nl} In order to enter the mine, you must save a young man name Vaidotas first. {nl}He was kidnapped and brought to the Vubbe outpost. {nl} Please, I'm begging for your help.
+QUEST_LV_0100_20150317_001563	$I have my roots deep in hunting. {nl} Don't worry about me go look after the others.
```

Comment:
Don't worry about me go look after the others. -> Don't worry about me. Go look after the others.